README.md

```diff
@@ -1,9 +1,13 @@
 # Valcheck ![Valcheck npm version](https://img.shields.io/npm/v/valcheck.svg)
 A NodeJS value checker.
 
-# Development
+## Development
 - run tests: `npm run test` (uses Mocha)
 <!-- - generate coverage report: `npm run coverage` (uses istanbul) -->
 <!-- - generate API documentation `npm run doc` (uses JSDoc) -->
 - run code style checker: `npm run lint` (uses TSLint)
 
+## Versioning and publishing
+Versioning is handled by bump2version.
```

Comment:
this deserves some more in-depth documentation (maybe in a wiki)

Reply:
yup will build a full documentation for nexus3 versioning + npm and docker repositories

Reply:
https://linkurious.atlassian.net/browse/OPS-72